lib/src/models/cvote_public_key.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:ledger_cardano/src/utils/constants.dart';
+import 'package:ledger_cardano/src/utils/validation_exception.dart';
+
+part 'cvote_public_key.freezed.dart';
+
+
+@freezed
+class CVotePublicKey with _$CVotePublicKey {
+  const CVotePublicKey._();
+
+  const factory CVotePublicKey({
+    required String value,
+  }) = CVotePublicKeyData;
```

Comment:
This should be renamed to `_CVotePublicKey`. See comment on `flags.dart` for explanations.